README.md

```diff
@@ -1,6 +1,6 @@
 # localtunnel
 
-[![Build Status](https://travis-ci.org/localtunnel/localtunnel.svg?branch=master)](https://travis-ci.org/localtunnel/localtunnel)
+[![Build Status](https://travis-ci.org/localtunnel/localtunnel.svg?branch=master)](https://travis-ci.org/localtunnel/localtunnel) [![Known Vulnerabilities](https://snyk.io/test/github/localtunnel/localtunnel/badge.svg)](https://snyk.io/test/github/localtunnel/localtunnel)
```

Comment:
Why are you adding more badges in a PR that is just meant to update the dependency version?